.github/workflows/k8s-demo-broker.yml

```diff
@@ -16,20 +16,29 @@ jobs:
     steps:
       - name: Check out the repo
         uses: actions/checkout@v2
-      - name: Push Atlas-osb to Docker Hub
+      - name: Get Tag for the image
+        id: tag
+        run: |
+          branch=$(echo $GITHUB_REF | awk -F "/" '{print $3}')
+          echo "::set-output name=tag::demo-$branch"
+
+      - name: Push Atlas-osb to Registry
```

Comment:
why does this action perform push as well? My understanding is that it's only the release process that is supposed to do this?

Reply:
now it is a local registry